### ISSUE TITLE
[package] readline/8.0: Provide support for Visual Studio

### ISSUE THREAD

**Issue by jmarrec** [conan-io/conan-center-index#4400]
Readline does not support MSVC https://github.com/conan-io/conan-center-index/blob/8c69557071cf79e94a6807c7b44a101c843da6ea/recipes/readline/all/conanfile.py#L47-L48

`vcpkg` on the other hand provides readline (granted, it's an old one). I tracked down the vpckg PR that added it: https://github.com/microsoft/vcpkg/pull/3504/files, which led me to the repo that enabled MSVC support for readline 5, seems like a small amount of patching: https://github.com/lltcggie/readline/commit/ea414b4e98475e3976198738061824e8a8379a50

It would be great if something similar could be achieved for conan!

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **readline/8.0**
  * Operating System+version: **Windows 10**
  * Compiler+version: **MSVC 2019**
  * Conan version: **conan 1.33.0**
  * Python version: **Python 3.8.3**

**Comment by prince-chrismc** [comment]
I like that vcpkg just made a new package for the fork with windows support.

I think this is a Library Request 👍

**Comment by perseoGI** [comment]
Hi @jmarrec!
Sorry for the delay, do you still need support for Visual Studio in this recipe?

**Comment by jmarrec** [comment]
My use case was ruby 2.7.2. Ruby > 3 (3.2.x I think) do not requires it, and I've transitioned recently to that version, so it's less of a concern to me, but would probably be a good idea nonetheless.

TL;DR: It'd be nice, but if that's unpratical I'm fine if you just want to close this issue.

**Comment by perseoGI** [comment]
Thank you for your quick and well formed response!
We do not have that much bandwidth right now so I'll close the issue for now and if in the future someone requests `readline` to be compilable with `Visual Studio` we will address this ASAP.

Thank you very much and happy coding! 🐸
